### PR TITLE
Fixes #14331 - Id param for content view puppet modules create not wo…

### PR DESCRIPTION
**Pull request by johnpmitsch**
…rking

### CODE REVIEW

**Comment by johnpmitsch** [comment]
To  test:
```hammer content-view puppet-module add --organization-id 1 --content-view-id CONTENT_VIEW_ID --id PUPPET_MODULE_ID```

check the api call on POST /content_views/:content_view_id/content_view_puppet_modules and the output of ```hammer -r content-view puppet-module list --organization-id 1 --content-view-id CONTENT_VIEW_ID```

the uuid column should have the puppet module uuid, not an active record id

File: app/controllers/katello/api/v2/content_view_puppet_modules_controller.rb [hunk 1]
```diff
@@ -24,7 +24,10 @@ def index
     param :id, String, :desc => N_("the id of the puppet module to associate")
     param :uuid, String, :desc => N_("the uuid of the puppet module to associate")
     def create
-      params[:content_view_puppet_module][:uuid] = @puppet_module.uuid if @puppet_module
+      if @puppet_module
+        params[:content_view_puppet_module][:uuid] = @puppet_module.uuid
+        params[:uuid] = @puppet_module.uuid
```
Comment (daviddavis) [inline app/controllers/katello/api/v2/content_view_puppet_modules_controller.rb]: It's generally a bad idea to modify params.

Comment (johnpmitsch) [inline app/controllers/katello/api/v2/content_view_puppet_modules_controller.rb]: It looks like we already have been modifying params to make sure a uuid properly passed, I would argue that this is doing the exact same thing.

Comment (daviddavis) [inline app/controllers/katello/api/v2/content_view_puppet_modules_controller.rb]: Wouldn't it be pretty easy to fix it though?

**Comment by johnpmitsch** [comment]
API is working fine - This is fixed by https://github.com/Katello/hammer-cli-katello/pull/368

**Pull request closed without merging**